pkgs/desktops/gnome/extensions/guillotine/default.nix

```diff
@@ -0,0 +1,54 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitLab,
+  glib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-guillotine";
+  version = "24";
+
+  src = fetchFromGitLab {
+    owner = "ente76";
+    repo = "guillotine";
+    rev = "v${version}";
+    hash = "sha256-eNhK3h9luPGXHR3lPkfu/mUN9+ixma64rbCk0cjF4Fc=";
+  };
+
+  nativeBuildInputs = [ glib ];
+
+  dontBuild = true;
+
+  passthru = {
+    extensionUuid = "guillotine@fopdoodle.net";
+    extensionPortalSlug = "guillotine";
+  };
+
+  preInstall = ''
+    rm schemas/gschemas.compiled
+    glib-compile-schemas schemas
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/gnome-shell/extensions/guillotine@fopdoodle.net
+    cp -R schemas "$out/share/gnome-shell/extensions/guillotine@fopdoodle.net"
+    cp default.json $out/share/gnome-shell/extensions/guillotine@fopdoodle.net
+    cp extension.js "$out/share/gnome-shell/extensions/guillotine@fopdoodle.net"
+    cp guillotine-symbolic.svg "$out/share/gnome-shell/extensions/guillotine@fopdoodle.net"
+    cp LICENSE "$out/share/gnome-shell/extensions/guillotine@fopdoodle.net"
+    cp metadata.json "$out/share/gnome-shell/extensions/guillotine@fopdoodle.net"
+    cp README.md "$out/share/gnome-shell/extensions/guillotine@fopdoodle.net"
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Guillotine is a gnome extension designed for efficiently carrying out executions of commands from a customizable menu. Simply speaking: it is a highly customizable menu that enables you to launch commands and toggle services.";
+    homepage = "https://gitlab.com/ente76/guillotine/";
+    license = licenses.gpl3Plus;
+    maintainers = with lib.maintainers; [ ];
```

Comment:
Not adding yourself?

Reply:
I wasn't sure what being a maintainer entailed, but it seems reasonable, I added myself